README.md

```diff
@@ -157,7 +157,7 @@ Put the button in the normal state.
 
 Put the button in the success state. Call the callback or the onSuccess prop when going back to the normal state.
 
-##### error([callback])
+##### error([callback, dontGoBackToNormal])
```

Comment:
I think I'd rather have a prop to control that. Something like `props.stayInErrorState` which would default to `false`.

Otherwise you need to handle it as an optional argument (eg. right now it would break when calling l.141)

Reply:
Yes true I actually made changes afterwards to avoid the error when calling `handlePromise` but forgot to push them